Neos.Utility.ObjectHandling/Classes/TypeHandling.php

```diff
@@ -51,9 +51,9 @@ public static function parseType(string $type): array
             throw new InvalidTypeException('Found an invalid element type declaration. A type "' . var_export($type, true) . '" does not exist.', 1264093630);
         }
 
-        $typeWithoutNull = self::stripNullableType($matches['type']);
-        $isNullable = $typeWithoutNull !== $matches['type'];
-        $type = self::normalizeType($typeWithoutNull);
+        $typeWithoutNull = self::stripNullableType($type);
```

Comment:
$typeWithoutNull/$isNullable never worked as expected - the nullable hint is not included in `$matches['type']`